packages/site/src/web/index.js

```diff
@@ -31,7 +31,8 @@ require('react-live/react-live.css');
 // Enable Glamor simulate helper
 simulations(true);
 
-// Github 404 page hack to support SPA hosting
```

Comment:
Related, I can't imagine that `site/public/404.html` still needed.  That file is a github-pages specific thing.